bug(client-go): error deserializing json

Running `deis config:list` in one application I see this error:
```
panic: interface conversion: interface is float64, not string

goroutine 1 [running]:
github.com/deis/deis/client/cmd.ConfigList(0x8206ee123, 0x12, 0x8206f1c00, 0x0, 0x0)
	/Users/matt/Projects/src/github.com/deis/deis/client/cmd/config.go:50 +0x7d7
github.com/deis/deis/client/parser.configList(0x8206e63b0, 0x1, 0x1, 0x0, 0x0)
	/Users/matt/Projects/src/github.com/deis/deis/client/parser/config.go:67 +0x27a
github.com/deis/deis/client/parser.Config(0x8206e63b0, 0x1, 0x1, 0x0, 0x0)
	/Users/matt/Projects/src/github.com/deis/deis/client/parser/config.go:24 +0x17d
main.Command(0x8206e63b0, 0x1, 0x1, 0x186840)
	/Users/matt/Projects/src/github.com/deis/deis/client/deis.go:89 +0x129b
main.main()
	/Users/matt/Projects/src/github.com/deis/deis/client/deis.go:18 +0x60
```

This application contains one particular configured value: `REDIS_PORT=6381`

--- a/client/cmd/config.go
+++ b/client/cmd/config.go
@@ -47,7 +47,7 @@ func ConfigList(appID string, oneLine bool) error {
 
 		// config.Values is type interface, so it needs to be converted to a string
 		for _, key := range keys {
-			configMap[key] = config.Values[key].(string)
+			configMap[key] = fmt.Sprintf("%v", config.Values[key])
 		}
 
 		fmt.Print(prettyprint.PrettyTabs(configMap, 6))

--- a/client/cmd/limits.go
+++ b/client/cmd/limits.go
@@ -30,7 +30,7 @@ func LimitsList(appID string) error {
 		memoryMap := make(map[string]string)
 
 		for key, value := range config.Memory {
-			memoryMap[key] = value.(string)
+			memoryMap[key] = fmt.Sprintf("%v", value)
 		}
 
 		fmt.Print(prettyprint.PrettyTabs(memoryMap, 5))

--- a/client/cmd/tags.go
+++ b/client/cmd/tags.go
@@ -25,7 +25,7 @@ func TagsList(appID string) error {
 	tagMap := make(map[string]string)
 
 	for key, value := range config.Tags {
-		tagMap[key] = value.(string)
+		tagMap[key] = fmt.Sprintf("%v", value)
 	}
 
 	fmt.Print(prettyprint.PrettyTabs(tagMap, 5))

